### DISCUSSION TITLE
Broken Image

### DISCUSSION THREAD

**Discussion by mpeck44** [learningeconomy/LearnCard#172] [Help] [answered]
gm gm 

Deployment of the credentials went real smooth.  Bot config, badge setup, and distribution worked well.  Only issue we ran into is the image attached to our badge show a broken link.

**Comment by Custard7** [comment]
> gm gm
> 
> Deployment of the credentials went real smooth. Bot config, badge setup, and distribution worked well. Only issue we ran into is the image attached to our badge show a broken link.

@mpeck44 - terrific to hear the deployment went smooth! 🎉  

We can definitely help you sort this, I'll just need some more info to help troubleshoot.

- Can you describe where you are seeing the broken image- in Discord, in LearnCard app, ... both?
- What's the name of your Credential?
- What is the link the your image?

Any other context would be helpful 💪

**Comment by Custard7** [comment, marked as answer]
For others who stumble on this thread, the issue was that the poster was using a "share link" from a Google drive, which does not actually link to the raw image file. Although you can find the raw image link from a google drive folder, it is fairly difficult, and we recommend using a more persistent, decentralized hosting service, such as NFT.Storage:

Right now, the Bot doesn't natively support hosting an image for your credential. Although you can host your image anywhere you would like, a great option is [NFT.Storage](https://nft.storage/files/) 

Upload the image into [NFT.Storage](https://nft.storage/files/), then click “Actions” on your uploaded image and click “View URL”—that’s the URL you want! It should look like: `https://<cid>.ipfs.nftstorage.link/`

Read more [here](https://docs.learncard.com/learncard-services/discord-bot#2-create-your-first-credential-template).
